siad/update.go

```diff
@@ -31,50 +31,50 @@ func newerVersion(version string) bool {
 		// inputs are trusted, so no need to check the error
 		ni, _ := strconv.Atoi(nums[i])
 		Ni, _ := strconv.Atoi(NUMS[i])
-		if ni > Ni {
-			return true
+		if ni != Ni {
+			return ni > Ni
 		}
```

Comment:
I don't like them either. Probably the worst variable names I've ever used.

I think my preference stems from the fact that I tend to write shorter functions, where it's easier to infer a variable's meaning from context. This is especially true in helper functions where the arguments can be generic. e.g.:
```
func greater(first, second int) bool { return first > second }
// is this really better than:
func greater(a, b int) bool { return a > b }
```
My functions also just tend to be less variable-heavy (possible due to my preference for literals), so there are fewer things to keep track of.